content/md/permissions-settings/access-rights-on-imports-exports_fr.md

```diff
@@ -0,0 +1,43 @@
+---
+id: access-rights-on-imports-exports
+themes: permissions-settings
+title: Définir les droits sur les **imports et exports**
+popular: false
+ee-only: true
+related: what-is-a-user-group, what-is-a-user, build-your-user-groups
+priority: low
+---
+
+# Introduction
+
+Vous pouvez facilement limiter l’édition ou l’exécution de profils d'[import](imports.html) et d'[export](exports.html) en fonction des [groupes d’utilisateurs](what-is-a-user-group.html) définis dans votre PIM.
+
+## Droits disponibles
+
+Il existe deux droits que vous pouvez accorder sur chaque profil d'import / export :
+- Le droit de l’exécuter nommé `Autorisé à exécuter les profils` : s'il est accordé, les utilisateurs du groupe d’utilisateurs pourront lancer le profil.
+- Le droit de le modifier nommé `Autorisé à éditer les profils` : s'il est accordé, les utilisateurs du groupe d'utilisateurs pourront modifier sa configuration, c'est-à-dire qu'ils auront accès au mode d'édition du profil d'import / export.
+
+# Définir les droits sur un profil d'import
+
+1. Allez dans `Imports`
+2. Sélectionnez dans la grille le profil d'import à configurer.
+3. Cliquez sur le bouton `Modifier`
+4. Cliquez sur l'onglet `Permissions`
+5. Cliquez sur les champs et sélectionnez des groupes d'utilisateurs pour leur accorder des autorisations.
+      ![image](Imports_Permissions_fr.png)
```

Comment:
Can you resize the image to a width of 840px max? It will avoid any resizing done by the browsers, which blur the images and make them unreadable. ;)

Reply:
Ok, i will have to do it for all the images I uploaded.

Reply:
Thanks!
It's documented in this [page](https://github.com/akeneo/pim-helpcenter/wiki/Write-a-wonderful-article#images), but I think it's not visible enough. I'll try to find a way to make it clearer.

Reply:
Modification done. The error comes from me not from the documentation.
Tell me if the modification is ok for you